lib/v2/gov/nifdc/index.js

```diff
@@ -0,0 +1,83 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const timezone = require('@/utils/timezone');
+const { parseDate } = require('@/utils/parse-date');
+
+module.exports = async (ctx) => {
+    const params = ctx.path === '/nifdc' ? '/nifdc/nifdc/bshff/ylqxbzhgl/qxggtzh' : ctx.path;
+
+    const rootUrl = 'https://www.nifdc.org.cn';
+    const currentUrl = `${rootUrl}${params.replace(/^\/nifdc/, '')}/index.html`;
```

Comment:
Why not use
```suggestion
    const params = ctx.path === '/nifdc' ? '/nifdc/bshff/ylqxbzhgl/qxggtzh' : ctx.path;

    const rootUrl = 'https://www.nifdc.org.cn';
    const currentUrl = `${rootUrl}${params)}/index.html`;
```
instead?

so we can have
```
/gov/nifdc/bshff/ylqxbzhgl/qxzqyj
/gov/nifdc/bshff/ylqxbzhgl/qxggtzh
/gov/nifdc/bshff/ylqxbzhgl/qxxxgk
```

Reply:
Since all urls start with `nifdc`...
It makes sense to shorten the routes 😃